packages/terra-clinical-result-view/src/ResultView.jsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconDocuments from 'terra-icon/lib/icon/IconDocuments';
+import IconComment from 'terra-icon/lib/icon/IconComment';
+import IconModified from 'terra-icon/lib/icon/IconModified';
+import IconUnexpected from 'terra-icon/lib/icon/IconUnexpected';
+
+import IconAbnormal from 'terra-icon/lib/icon/IconAbnormal';
+import IconAlert from 'terra-icon/lib/icon/IconAlert';
+import IconHigh from 'terra-icon/lib/icon/IconHigh';
+import IconLow from 'terra-icon/lib/icon/IconLow';
+import styles from './ResultView.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   *  An array of one or more results.
+   */
+  results: PropTypes.arrayOf(PropTypes.shape({
+    value: PropTypes.number,
+    units: PropTypes.string,
+    normalcy: PropTypes.string,
+  })),
+  /**
+   *  The age display of the result. Does not handle formatting.
+   */
+  timeDisplay: PropTypes.string,
+  /**
+   *  Does the result have related documents. Adds an indicator.
+   */
+  hasDocument: PropTypes.bool,
+  /**
+   *  Has the result been modified. Adds an indicator.
+   */
+  isModified: PropTypes.bool,
+  /**
+   *  Are there comments related to the result. Adds an indicator.
+   */
+  hasComment: PropTypes.bool,
+  /**
+   *  Is the view aligned to the right. Defaults to false.
+   */
+  alignEnd: PropTypes.bool,
+  /**
+   *  Can the result text be truncated. Defaults to true.
+   */
+  isTruncated: PropTypes.bool,
+  /**
+   *  Should the view have padding. Defaults to false.
+   */
+  isPadded: PropTypes.bool,
+};
+
+const defaultProps = {
+  results: [],
+  timeDisplay: null,
+  hasDocument: false,
+  hasComment: false,
+  isModified: false,
+  alignEnd: false,
+  isTruncated: true,
+  isPadded: false,
+};
+
+const normalcyIndicators = {
+  ABNORMAL: <IconAbnormal />,
```

Comment:
On second thought, do we know if the normalcy indicators are constrained to these four icons, or should we be agnostic on what the indicators are? If that's the case, we could change the `normalcy` prop to a type `node`, and let the consumer pass in whatever icon they want. Thoughts?

Reply:
We should be agnostic on the indicators and rgb values in the terra-clinical-result-view. From my investigation I found the FHIR spec for observation has quite a large set of what can be considered normalcy. https://www.hl7.org/fhir/valueset-observation-interpretation.html